packages/static-module-record/test/test-static-module-record.js

```diff
@@ -588,7 +588,8 @@ test('import meta present', t => {
   t.is(record.__needsImportMeta__, true);
 });
 
-test('export names', t => {
+// FIXME: this test depends on published SES.
+test.failing('export names', t => {
```

Comment:
Sadly two tests here include stable SES behavior in the test. I tried replacing SES locally, but didn't succeed.

Reply:
`compartment-mapper` tests have a thing where they build ses before running.
might be complicated dep topology bc it uses the code from compartment-mapper

Reply:
does just going into ses and running `yarn build` not resolve the issue locally?

Reply:
Nope, I tried that.

Reply:
Ok, some experience from the bundler helped me notice the test is not using notifiers mapping from ses, but instead implementing its own. fixing.

Reply:
aaaand fixed